test/aliceO2-namespace-naming.cpp

```diff
@@ -0,0 +1,152 @@
+// RUN: %check_clang_tidy %s aliceO2-namespace-naming %t
+
+namespace FirstNamespace
+// CHECK-MESSAGES: :[[@LINE-1]]:11: warning: namespace 'FirstNamespace' does not follow the underscore convention [aliceO2-namespace-naming]
+// CHECK-FIXES: {{^}}namespace first_namespace{{$}}
+{
+  int a;
+  void fooFromNamespace();
+  
+  void bar()
+  {
+  }
+  
```

Comment:
Correct examples are checked when you don't write any annotation "CHECK-MESSAGE" or "CHECK-FIXES". Meaning you don't expect to see any changes in that line of code. If a change occurs(meaning the check saw a problem in that line and throws a warning or reworks the line of code), then the test fails by saying that unexpected change is seen.